### PR TITLE
Initialise sysl.io website

### LINKED ISSUE #125: Setup basic HUGO website and create deployment

### PR DESCRIPTION
**Pull request by juliaogris**
Fixes #125.

Changes proposed in this pull request:
- Update `sysl --help` to hide `deps` command
-  Move `gradle` docs into `environment guide`
- Create a seed for the sysl.io website :
   *  [current version](https://nostalgic-jones-2dabea.netlify.com/) 
   * [old website](https://anz-bank.github.io/sysl-website/).

Outstanding work:
* Write [Command line docs](https://nostalgic-jones-2dabea.netlify.com/docs/commandline/)
* Write [Sysl syntax docs](https://nostalgic-jones-2dabea.netlify.com/docs/syntax/)
* Write [Output formats docs](https://nostalgic-jones-2dabea.netlify.com/docs/outputs/)

@anz-bank/sysl-developers

### CODE REVIEW

**Comment by juliaogris** [comment]
@sahejsingh , @anzdaddy / @marcelocantos - can you please review this?

File: docs/environment_guide.md [hunk 1]
```diff
@@ -15,8 +15,38 @@ If you are behind an NTLM proxy, you might need to setup `cntlm` to work effecti
 In case of SSL certificate interception you might need to add `--trusted-host pypi.python.org` to your pip commands or setup a global `pip.conf`
 file (see [official docs](https://pip.pypa.io/en/stable/user_guide/#config-file) or [Stackoverflow](https://stackoverflow.com/a/46410817)).
 
+Gradle setup in coporate environment
```
Comment (sahejsingh) [inline docs/environment_guide.md]: coporate -> corporate.

File: docs/ideas/root.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Sysl root
+=========
+
+When executing `sysl` one needs to specify a root directory with `--root DIR`, which serves as root for specifying `sysl modules` on the command line and in import statement:
+
+* `sysl textpb --rot sysl_dir /town`
```
Comment (sahejsingh) [inline docs/ideas/root.md]: --root.

File: docs/ideas/root.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* `import /transit/bus` at to of town.sysl file
```
Comment (sahejsingh) [inline docs/ideas/root.md]: did you mean
at top of town.sysl ?

File: docs/ideas/root.md (hunk 2 above)
Comment (juliaogris) [reply docs/ideas/root.md]: Done.

File: docs/ideas/root.md (hunk 3 above)
Comment (juliaogris) [reply docs/ideas/root.md]: I did, thanks, done.

**Pull request merged**